web_hidden_field/models/ir_ui_view.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Ignacio Ibeas <ignacio@acysos.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import api, models
+from odoo.osv import orm
+import json
+
+
+class IrUiView(models.Model):
+    _inherit = 'ir.ui.view'
+
+    @api.multi
+    def _check_hidden_field(self, model_name, field_name):
+        model = self.env['ir.model'].search([('model', '=', model_name)])
+        field = self.env['ir.model.fields'].search(
+            [('name', '=', field_name), ('model_id', '=', model.id)])
+        hidden_field = self.env['hidden.template.field'].search(
```

Comment:
you can do one single query, no need to search model and field:
```python
domain = [
    ('field_id.name', '=', field_name), 
    ('model_id.name', '=', model_name),
    [...]
]
```
Note that I changed field names according to my comments above.